File: 01-Ejercicios-Clases/Proyecto-Curso/ProyectosJAVA/LP2SoftRRHHController/src/pe/edu/pucp/lp2soft/rrhh/dao/AreaDAO.java
package pe.edu.pucp.lp2soft.rrhh.dao;
import java.util.ArrayList;
import pe.edu.pucp.lp2soft.rrhh.model.Area;
public interface AreaDAO {
    int insertar(Area area);
    int modificar(Area area);
    int eliminar(int idArea);
    ArrayList<Area> listarTodas();
}

File: 04-Proyecto-LP2Soft/ProyectosNET/LP2Soft/LP2SoftRRHHController/DAO/AreaDAO.cs
using LP2SoftRRHHModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftRRHHController.DAO
{
    public interface AreaDAO
    {
        int insertar(Area area);
        int modificar(Area area);
        int eliminar(int idArea);

        BindingList<Area> listarTodas();
    }
}
